sql-odbc/src/sqlodbc/dlg_specific.h

```diff
@@ -120,7 +120,7 @@ extern "C" {
 #define DEFAULT_BOOLSASCHAR 0
 #define DEFAULT_UNIQUEINDEX 1 /* dont recognize */
 #define DEFAULT_LOGLEVEL OPENSEARCH_WARNING
-#define DEFAULT_USE_SSL 0
+#define DEFAULT_USE_SSL 1
```

Comment:
Is there any doc should be change accordingly?

Reply:
I could not find any reference to a default value for the use of SSL. We will make a ticket to update this page with at least a reference to download the driver.
https://opensearch.org/docs/latest/search-plugins/sql/odbc/

Reply:
And also to note the defaults for the ODBC driver.